app/models/file_depot_ftp.rb

```diff
@@ -93,12 +93,15 @@ def file_exists?(file_or_directory)
   private
 
   def create_directory_structure(directory_path)
-    Pathname.new(directory_path).descend do |path|
```

Comment:
@isimluk Please ask around and let me know if you can't get access for testing.  Also, if you have ideas on how to capture this behavior so we can automate tests to avoid regression, I'd love to hear any ideas.

I'm fine with the change but would want some sanity checks on various ftps.   Maybe @jcarter12 can DM you some information to test against the ftp they use for anonymous access.